docs/7-dashboard-ui/3-Zarf-deployment-ui.md

```diff
@@ -0,0 +1,115 @@
+# Zarf Deployment Web UI
+
+Zarf has a Deoployment Web UI built in. The Deployment Web Ui supports a nubmer of Zarf features used during the package deployment process. For users who prefer not to use the command line tool, the Web UI creates a simple experience to deploy and manage Zarf clusters and packages. The Web UI can be used to connect to connect to existing clusters (via Kubeconfig), initalize a cluster, deploy packages into a cluster, update packages in the cluster, and remove packages from the cluster. 
+
+The Zarf Web UI mirros the functionality of the Zarf CLI commands, but with a more intuitive flow and familuar web application patterns for non-technical users. The web UI does not offer any additional commands or core functionaliity to Zarf. 
+
+## Open the Zarf Deployment Web UI
+
+The Zarf Deployment Web UI can be easily spun up with a single command from the CLI terminal. 
+
+Follow these steps to get started using the Web UI
+
+1. Step one: Install Zarf binary
+2. Step two: Open CLI terminal
+3. Step three: Type in the following command: ```zarf dev UI```
+
+![GIF showing the Web UI lauch from the CLI terminal](../.images/dashboard/Web_UI__Launch_w__Cluster_AdobeExpress.gif)
+
+## Using Zarf Deployment Web UI
+
+### Cluster Connection Status
+
+When Zarf is running it automatically searches for a Kubeconfig on the local machine. If the Kubeconfig is found, it searches the defualt cluster to determine if it is a Zarf cluster (Initialized). There are two different cluster statuses the Web UI will display based on the state of the cluster found. 
+
+#### Cluster not Connected (Not Initizalized)
+
+![Web UI shows organge warning status and message "cluster not connected" on the cluster card](../.images/dashboard/Web%20UI%20-%20Cluster%20Not%20Connected.png)
+
+1. Shown when there is no Kubeconfig found on the machine.
+2. Shown when there is a Kubeconfig foudn on the machine but it has not currently connected to Zarf.
```

Comment:
"connect to Zarf" is kind of a weird way to put it - I think it might be fine for less technical people but _technically_ it is reaching out and looking for a few Kubernetes secrets that hint/suggest Zarf is deployed / setup in the cluster - more like a tracker that sees footprints and gathers information from that rather than talking to the person that left those footprints.

Reply:
Hmmmm - I am ok with saying both. I think one way to educate less technical users is use familiar language then explain what we mean technically by it